docs/adapters/development.md

```diff
@@ -94,5 +94,11 @@ There is a an open issue in the node community around [npm linked peer dependenc
   }
   ```
 3. Now try running `hubot -a sample` again and see that the imports are properly loaded.
-4. Once this is working properly, you can build out the functionality of your adapter as you see fit.  Take a look at some of the other adapters to get some ideas for your implementation.
+4. Once this is working properly, you can build out the functionality of your adapter as you see fit. Take a look at some of the other adapters to get some ideas for your implementation.
   - Once packaged and deployed via `npm`, you won't need the dependency in `hubot` anymore since the peer dependency should work as an official module.
+
+# Or, just load the adapter file from local disk
```

Comment:
Suggest to remove "just" here, often intended to represent a step as small, but covers for minimising complexity in documentation.